shared/js/background/newtab-tracker-stats.js

```diff
@@ -61,10 +66,12 @@ export class NewTabTrackerStats {
     ports = []
 
     /**
-     * @param {import("../background/classes/tracker-stats").TrackerStats} stats - the interface for the stats data.
+     * @param {TrackerStats} stats - the interface for the stats data.
+     * @param {Settings} extensionSettings
      */
-    constructor (stats) {
+    constructor (stats, extensionSettings) {
```

Comment:
I think it makes more sense to just import settings in this file, rather than to plumb it through.